tool/templates/cpp/enum_decl.h.jinja

```diff
@@ -3,6 +3,11 @@
 {% if let Some(ns) = namespace -%}
 namespace {{ns}} {
 {% endif -%}
+{% if !docs.is_empty() -%}
+/**
+ * {{docs}}
```

Comment:
nit(optional): id prefer if the choice of how the `*` gets handled were done with a filter applied here, rather than in the common `fmt_docs` function. Probably define an `add_doc_asterisks` somewhere.

Reply:
what choice exactly?